packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -116,4 +128,62 @@ contract L1Block is ISemver {
             sstore(batcherHash.slot, calldataload(132)) // bytes32
         }
     }
+
+    /// @notice Updates the L1 block values for an Interop upgraded chain.
+    /// @dev The interop upgrade block itself MUST include a call to setL1BlockValuesEcotone.
+    /// Params are packed and passed in as raw msg.data instead of ABI to reduce calldata size.
+    /// Params are expected to extend setL1BlockValuesEcotone calldata in the following order:
+    ///  1. _interopSetSize     Size of the interop dependency set.
+    ///  2. _chainIds           Array of chain IDs in the interop dependency set.
+    function setL1BlockValuesInterop() external {
```

Comment:
This parses the calldata created in https://github.com/ethereum-optimism/optimism/pull/9840 so it needs to handle setting the rest of the state too